application/models/DbTable/CountrieShipmentMap.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class Application_Model_DbTable_CountrieShipmentMap extends Zend_Db_Table_Abstract {
```

Comment:
The file name and class name are misspelled. They should be:
CountryShipmentMap.php
and
Application_Model_DbTable_CountryShipmentMap